documentation/swagger-accuracy-report.md

```diff
@@ -58,7 +70,5 @@ For any `Untested Operations`, you must modify the RESTler configuration files t
 
       >Restler.exe test --dictionary_file dict.json --grammar_file grammar.py --settings engine_settings.json --token_refresh_command "/local-scripts/get-token.sh" --token_refresh_interval 60
 
-
- 3. Upload the modified conf file so the PR pipeline can use it and re-run the Service Contract Accuracy report
+3.  Upload the modified conf file so the PR pipeline can use it and re-run the Service Contract Accuracy report
```

Comment:
Please add this part too including the location to upload configs in the PR and how to re-run pipeline.